src/GraphQL/Execution/ExecutionStrategy.cs

```diff
@@ -178,35 +178,11 @@ protected virtual async Task<ExecutionNode> ExecuteNodeAsync(ExecutionContext co
             if (node.IsResultSet)
                 return node;
 
-            ResolveFieldContext resolveContext = null;
+            IResolveFieldContext resolveContext = null;
```

Comment:
Please remove `using static GraphQL.Execution.ExecutionHelper;`

Reply:
It is still in used by 7 other calls in this file.  I did remove excess usings in the two new files.